### PR TITLE
feature/connecting_SQLAlchemy_new

### PR DESCRIPTION
**Pull request by bannybaks**
Container with database Postrges ver. 15.3-alpine3.18

### CODE REVIEW

**Comment by bannybaks** [comment]
Installed and connected SQLALchemy; The function of creating a session is described

**Comment by bannybaks** [comment]
The base class for creating models is described

**Comment by bannybaks** [comment]
Of all the tasks, only Alembic is not connected

File: src/core/db/session.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from sqlalchemy.ext.asyncio import AsyncSession, create_async_engine
+from sqlalchemy.orm import sessionmaker
+
+DATABASE_URL = "postgresql+asyncpg://postgres:postgres@localhost:5432/postgres"
+
+engine = create_async_engine(DATABASE_URL, future=True, echo=True)
+session_factory = sessionmaker(engine, expire_on_commit=False, class_=AsyncSession)
```
Comment (NiKuma0) [inline src/core/db/session.py:7]: В новой версии это делается через `async_sessionmaker`

**Comment by bannybaks** [comment]
Module create session is corrected

**Review by NiKuma0 — APPROVED** [review]

**Pull request merged**